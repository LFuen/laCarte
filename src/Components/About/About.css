.aboutTitle {
    margin: 0 0 0 0;
    font-size: 40px;
}

.aboutPara {
    font-size: 20px;
    font-family: 'Sacramento', cursive;
    font-family: 'Special Elite', cursive;
    text-decoration: none;
    color: rgb(0, 0, 0);
    margin-top: 20px;
}

*, *:after, *:before {
    -webkit-box-sizing: border-box;
        -moz-box-sizing: border-box;
            box-sizing: border-box;
}

.quote {
    position: relative;
    letter-spacing: .03em;
    margin-bottom: .5rem;    
    line-height: 1.4em;
    font-size: 40px;
    color: black;
}

.quote:before {
        content: "“";
        position: absolute;
        left: -.7em;
}
    
.quote:after {
        content: "”";
        margin-right: -1rem;
}

.quote--container {
    margin: 4.5rem auto 0;
    width: 30em;
    padding-bottom: .5rem;
}

.quote--highlight {
    color: #D24335;
}

.quote--author {
    font-family: 'Open Sans Condensed';
    font-size: .8rem;
    text-align: right;
    font-weight: bold;
    color: black;
}

.about {
    width: 100%;
    margin: 1.5rem auto 100px;
}

.paragraph {
    background-color:rgb(233, 111, 111);
    border: 3px solid aliceblue;
    border-radius: 50px;
    display: flex;
    width: 65%;
    flex-direction: column;
    justify-content: center;
    margin-top: 10px;
    margin-left: auto;
    margin-right: auto;
    padding: 20px;
    color: aliceblue;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.4);
}

.available {
    font-size: 10px;
    margin-top: 20px;
    font-weight: bold;
}

.about {
    color:black !important;
}

