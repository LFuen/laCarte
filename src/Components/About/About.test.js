import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import LaCarteContext from "../../context/LaCarteContext";
import About from "./About";

describe(`About Component`, () => {
  describe(`Snapshot`, () => {
    it("renders the UI as expected", () => {
      const about = renderer.create(<About />).toJSON();
      expect(about).toMatchSnapshot();
    });
  });

  describe(`Smoke test`, () => {
    it(`Renders without crashing`, () => {
      const div = document.createElement("div");

      const value = {
        meals: "string",
        chefs: "string",
        cuisines: "string",
        orders: "string",
        addOrder: "string",
        updateOrder: "string",
        deleteOrder: "string",
      };

      ReactDOM.render(
        <LaCarteContext.Provider value={value}>
          <About />
        </LaCarteContext.Provider>,
        div
      );
      ReactDOM.unmountComponentAtNode(div);
    });
  });
});
