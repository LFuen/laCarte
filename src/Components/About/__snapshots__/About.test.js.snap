// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component Snapshot renders the UI as expected 1`] = `
<div>
  <div
    className="about"
  >
    <div
      className="quote--container"
    >
      <p
        className="quote"
      >
        We all 
        <span
          className="quote--highlight"
        >
          eat
        </span>
        … and it would be a sad waste of opportunity to eat 
        <span
          className="quote--highlight"
        >
          badly
        </span>
        .
      </p>
      <p
        className="quote--author"
      >
        – Anna Thomas
      </p>
    </div>
    <div
      className="paragraph"
    >
      <p
        className="aboutPara"
      >
        Face it: you're a workaholic. Maybe not always consciously, but more often than not, you look up at the end of that "last thing" you were just finishing up, and notice you're the only one 
        <em>
          still
        </em>
         in the office! Damn, not again! That's the 
        <em>
          third
        </em>
         time this week! Oh well, you think to yourself, no distractions ; I can finish this 
        <em>
          real
        </em>
        last thing before I go home and...DINNER! You haven't had dinner!
      </p>
      <p
        className="aboutPara"
      >
        This is where 
        <em>
          we
        </em>
         step in. No more fast, greasy, or generally unhealthy late-night entrees that conspire with your workaholic ways to cause that inevitable heart attack! We're not judging, just sayin'...
      </p>
      <p
        className="aboutPara"
      >
        Let our world-class and trained culinary chefs provide thoughtful and well-balanced meals for you so you can finally,
        <em>
          finally
        </em>
         get rid of that "last thing" that's taking precedence right now!
      </p>
      <p
        className="available"
      >
        ** Currently, we're only able to offer our services to the greater D.C. area. Please check back periodically as we add new cities!**
      </p>
    </div>
  </div>
</div>
`;
